### PR TITLE
Optimize data loading process disk usage

### PR DESCRIPTION
**Pull request by arkid15r**
## Issue Number
NA

## Purpose/Implementation Notes

Clean up output data for each project once it's processed.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests
NA

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots
NA

### CODE REVIEW

File: api/scpca_portal/management/commands/load_data.py [hunk 1]
```diff
@@ -80,14 +80,19 @@ def handle(self, *args, **options):
...
+        ComputedFile.README_SPATIAL_FILE_NAME,
+    )
     for item in cleanup_items:
         for path in Path(common.OUTPUT_DATA_DIR).glob(item):
             path.unlink()
 
 
 def load_data_from_s3(
     allowed_submitters: set = ALLOWED_SUBMITTERS,
+    cleanup_output_data=not (settings.DEBUG or settings.TEST),
```
Comment (davidsmejia) [inline api/scpca_portal/management/commands/load_data.py:95]:
```suggestion
    default_cleanup_output_data: bool = IS_PRODUCTION),
```

Comment (davidsmejia) [reply api/scpca_portal/management/commands/load_data.py:95]: Add to the top of the file, under `ALLOWED_SUBMITTERS`:
```python
IS_PRODUCTION = not (settings.DEBUG or settings.TEST)
```

Comment (davidsmejia) [reply api/scpca_portal/management/commands/load_data.py:95]: Also add another parameter that overrides the option `default_cleanup_output_data` so if we want to persist the data for debugging we can without changing the settings in production.

File: api/scpca_portal/management/commands/load_data.py [hunk 2]
```diff
@@ -203,3 +208,11 @@ def load_data_from_s3(
                     settings.AWS_S3_BUCKET_NAME,
                     computed_file.s3_key,
                 )
+
+        if cleanup_output_data:
```
Comment (davidsmejia) [inline api/scpca_portal/management/commands/load_data.py:212]: if you add the parameter to allow for overriding:
```suggestion
        if cleanup_output_data or default_cleanup_output_data:
```

**Review by davidsmejia — CHANGES_REQUESTED** [review]
Also you can scale the volume size to 100GB and test running everything available on staging. https://github.com/AlexsLemonade/scpca-portal/blob/dev/infrastructure/api.tf#L89

I would create a new ticket to make sure this gets tested.

File: api/scpca_portal/management/commands/load_data.py (hunk 1 above)
Comment (arkid15r) [reply api/scpca_portal/management/commands/load_data.py]: I've added the implementation based on your comments.

**Comment by arkid15r** [comment]
PTAL

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**